driver/pgdriver/listener.go

```diff
@@ -237,21 +237,30 @@ type Notification struct {
 
 type ChannelOption func(c *channel)
 
+type channelOverflowHandler func(n Notification)
+
 func WithChannelSize(size int) ChannelOption {
 	return func(c *channel) {
 		c.size = size
 	}
 }
 
+func WithChannelOverflowHandler(handler channelOverflowHandler) ChannelOption {
+	return func(c *channel) {
+		c.overflowHandler = handler
+	}
+}
```

Comment:
Will you be able to add a test case to `driver_test.go`?
Adding this option is a trivial change, but it'd be good to have that behavior tested.